opennlp-tools/src/main/java/opennlp/tools/cmdline/dictionary/DictionaryBuilderTool.java

```diff
@@ -56,8 +58,8 @@ public void run(String[] args) {
     CmdLineUtil.checkInputFile("dictionary input file", dictInFile);
     CmdLineUtil.checkOutputFile("dictionary output file", dictOutFile);
 
-    try (InputStreamReader in = new InputStreamReader(new FileInputStream(dictInFile), encoding);
-        OutputStream out = new FileOutputStream(dictOutFile)) {
+    try (Reader in = new BufferedReader(new InputStreamReader(new FileInputStream(dictInFile), encoding));
+         OutputStream out = new FileOutputStream(dictOutFile)) {
```

Comment:
No `BufferedOutputStream` for `FileOutputStream` ?

Reply:
This PR/issues targets _read_ IO operations, only. _Write_ is a separate topic.